docs/CHANGELOG/changelog-0.9.x.md

```diff
@@ -0,0 +1,79 @@
+# Claudie `v0.9`
+
+!!! warning "Due to changes to the core of how claudie works with terraform files and representation of the data in persistent storage the `v0.9.x` version will not be backwards compatible with clusters build using previous claudie versions."
+
+## Most notable changes (TL;DR)
+
+- Support for pluggable external terraform files was added, breaking the dependency of updated terraform files on a new claudie version. The ability to arbitrarily change the templates used by claudie was made available to the user. As a result, claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. 
```

Comment:
`s/claudie/Claudie/`
`s/hetzner/Hetzner/`
`s/ubuntu/Ubuntu/`

Throughout the whole doc.